ServerlessLibraryAPI/StorageHelper.cs

```diff
@@ -71,25 +71,26 @@ public static async Task<IEnumerable<SLItemStats>> getSLItemRecordsAsync()
             return entities;
         }
 
-        public async Task<SLItemStats> GetItem(string template)
+        public async Task<SLItemStats> GetItem(string id)
         {
-            TableOperation operation = TableOperation.Retrieve<SLItemStats>(template, template);
+            TableOperation operation = TableOperation.Retrieve<SLItemStats>(id, id);
 
             TableResult result = await (await getTableReference()).ExecuteAsync(operation);
 
             return (SLItemStats)(dynamic)result.Result;
         }
 
- }
+    }
 
     public class SLItemStats : TableEntity
     {
-        public string template { get; set; }
+        public string id { get; set; }
         public int totalDownloads { get; set; }
         public int downloadsToday { get; set; }
```

Comment:
are we still going to keep downloads today/this week/this month ? do we show them anywhere?

Reply:
In new design, not used. I am open to remove these.

Reply:
removed